package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "exec-buffer",
-  "version": "3.0.0",
+  "version": "3.0.1",
```

Comment:
Fixed in https://github.com/kevva/exec-buffer/commit/7733dd0cd3d4d46301a367a424dd9a2b395d61b2.